### ISSUE TITLE
v2.0.9

### ISSUE THREAD

**Issue by tara81** [EndlessCodeGroup/RPGInventory#49]
Problems: 

1 . /rpginv open playername
Doesn't let us view other player's inventory. If you type it , it just says "/rpginv open "

2. Players can no longer remove their armor. Tried out upgraded to v2.0.9 from 1.3.5

3. Players can take the border out of the rpginventory view. The border cannot be changed from Hoes or w/e it is , even though in the configuration I put glass panes.

4. Players cannot see their armor in /rpginv open. They can see it in regular equipment view only.

5. Players have to earn crafting levels to craft? wtf? wheres the config to turn this off??

In conclusion, this update just isn't viable if players cannot even equip armor in the rpg inventory GUI.

**Comment by OsipXD** [comment]
So strange, I haven't these problems.
Have you tried to run plugin on "clean" server?

**Comment by tara81** [comment]
I guess I will have to , this is very confusing if it works properly for you. I run a lot of plugins, and version 1.3.4 works perfectly with them so I dont understand why this update is borked

**Comment by tara81** [comment]
I am still on 1.9.4 if that matters.

**Comment by OsipXD** [comment]
Can you provide me your logs? (post it on https://pastebin.com/)

**Comment by tara81** [comment]
No errors though =( https://pastebin.com/Qdij0t2n

**Comment by OsipXD** [comment]
So strange... Ok, I'll try to reproduce it again

**Comment by OsipXD** [comment]
Is bug  still here on new versions?

**Comment by OsipXD** [comment]
No answer 1 month. Closed
